--- a/OpenEphyra/start_server.sh
+++ b/OpenEphyra/start_server.sh
@@ -1 +1,3 @@
+#!/bin/bash
+cd /usr/lib/lucida-openephyra
 java -Djava.library.path=lib/search/ -classpath bin:lib/learn/jsoup-1.8.3.jar:lib/ml/maxent.jar:lib/ml/minorthird.jar:lib/nlp/jwnl.jar:lib/nlp/lingpipe.jar:lib/nlp/opennlp-tools.jar:lib/nlp/plingstemmer.jar:lib/nlp/snowball.jar:lib/nlp/stanford-ner.jar:lib/nlp/stanford-parser.jar:lib/nlp/stanford-postagger.jar:lib/qa/javelin.jar:lib/search/bing-search-java-sdk.jar:lib/search/googleapi.jar:lib/search/indri.jar:lib/search/yahoosearch.jar:lib/thrift/libthrift-0.9.2.jar:lib/thrift/log4j-1.2.14.jar:lib/thrift/slf4j-api-1.5.8.jar:lib/thrift/slf4j-log4j12-1.5.8.jar:lib/util/commons-logging.jar:lib/util/gson.jar:lib/util/htmlparser.jar:lib/util/jetty-all.jar:lib/util/log4j.jar:lib/util/servlet-api.jar:lib/util/trove.jar:lib/db/mongo-java-driver-3.2.2.jar  lucida.main.QADaemon
\ No newline at end of file
--- a/OpenEphyra/Makefile
+++ b/OpenEphyra/Makefile
@@ -3,8 +3,8 @@
 
 thrift:
 	@if [ ! -d "src/lucida/thrift" ]; then \
-       thrift --gen java ../../lucidaservice.thrift; \
-       thrift --gen java ../../lucidatypes.thrift; \
+       thrift --gen java ../debian/lucidaservice.thrift; \
+       thrift --gen java ../debian/lucidatypes.thrift; \
        sed -i '1s/^/package lucida.thrift;\n/' gen-java/LucidaService.java; \
        sed -i '1s/^/package lucida.thrift;\n/' gen-java/QueryInput.java; \
        sed -i '1s/^/package lucida.thrift;\n/' gen-java/QuerySpec.java; \
--- a/Makefile
+++ b/Makefile
@@ -1,2 +1,2 @@
 SUBDIRS=OpenEphyra
-include ../../Makefile.common
\ No newline at end of file
+include ./debian/Makefile.common
